nextflow_schema.json

```diff
@@ -81,7 +81,14 @@
                 },
                 "orf_caller": {
                     "type": "string",
-                    "description": "Specify which ORF caller you would like to run."
+                    "description": "possible alternatives: prodigal, default: prodigal.",
```

Comment:
Well, you should have kept the first part, to make it "Specify which ORF caller you would like to run, possible alternatives: prodigal, default: prodigal."!